### DISCUSSION TITLE
[4.x]: Rerouting from saving as new from the image editing tool (Craft 3 and 4)

### DISCUSSION THREAD

**Discussion by JeanLucEsser** [craftcms/cms#10977] [Ideas]
### Description

This does also apply to Craft 3.

When saving as a new image from the Craft image edition tool, we're sent back to the original image we were editing. Maybe that's intended but I think we can expect to be sent to newly created one. So we can edit the name and other properties.

Right now many clients and users are editing myImage (A), saving as new myImage (B), changing its name to myNewImage only to realize they changed the name of A expecting to change the name of B.

Making matters worse, the name of the asset stays exactly the same when saving as new. When changes are minimal, it makes impossible to make which is which in the assets list. We have to rely on the updated at column.

### Expected behavior

Edit myImage.jpg (A)
Make small changes (a little crop)
Save as new image (B)
We are sent to the edit window of (B)
The name of the image is myImage - copy 01.jpg
We change the name to myNewImage

### Actual behavior 01

Edit myImage.jpg (A)
Make small changes (a little crop)
Save as new image (B)
We are sent to the edit window of (A)
We change the name to myNewImage
Oops, we changed the name of the wrong image

### Actual behavior 02

Edit myImage.jpg (A)
Make small changes (a little crop)
Save as new image (B)
We are sent to the edit window of (A)
Looks like we're still on (A)
We go back to the list af assets
Mmm, looks like both images have the same name
Can't figure out which one is the new based on the thumbnail
Worst case scenario : Oops, the column updated at has been hidden by the admin
...

### Craft CMS version

4.0

### Installed plugins and versions

-

**Comment by brandonkelly** [comment]
This is currently working as expected, so converted the issue to a discussion. 

Worth mentioning that Assets fields will now automatically relate the new asset when you got to “Save as a new asset” from the field. (#8974)

**Reply by thupsi** [reply]
Well, there is no right or wrong in this matter, but I agree with @JeanLucEsser that the expected (by the user) behavior when the user saves "as new" would be to continue editing the new asset. Pretty much every app/software I can think of behaves like this. 

I do think that both behaviors are valid, since they cater to distinct workflows, but the way it works now seems (to me) less useful for the majority of situations.
